docs/persist.rst

```diff
@@ -55,7 +55,13 @@ Rasa NLU supports using `S3 <https://aws.amazon.com/s3/>`_ and
 
     If there is no container with the name ``AZURE_CONTAINER`` Rasa will create it.
 
-Models are gzipped before saving to cloud.
+Models are gzipped before they are saved in the cloud. The gzipped file naming convention
+is `{PROJECT}___{MODEL_NAME}.tar.gz` and it is stored in the root folder of the storage service.
```

Comment:
@careless25 Sure it is not `{PROJECT}{MODEL_NAME}.tar.gz`?

Reply:
Yes, see lines 96-100 in persistor.py

Reply:
👍 You know the code better than I - awesome job!